README.md

```diff
@@ -101,6 +101,19 @@ All of the roles internally have the notion of the following "steps" that are co
 
 Running a playbook will run the respective roles for each of these. If instead you want to run a subset you can use the operation argument as such `ansible-playbook -e operation=run`.
 
+#### JFR Hyperfoil
+
+To record Hyperfoil into JFR files we need to enable the options. It is possible to record on the controller and the agents separately, which with a different approach.
+To collect on the controller, when running the controller benchmark to start, pass the `-e hyperfoil_jfr=true` argument.
+
+The agents are started by the controller. Therefore, we need to provide the custom JVM arguments, we can do it when running the agent playbook. This is done with `-e agent_java_args="'-Xmx800m -XX:+FlightRecorder -XX:StartFlightRecording=compress=false,delay=10s,duration=24h,settings=profile,filename=/tmp/hyperfoil/workspace/hfa-record.jfr'"` argument.
```

Comment:
The java_args here are all optional and the filename is only required if `-e download_jfr=true` is also specified. My point being is that we may want to explain they can use any flight recorder settings but just the filename has to be very specific if downloaded.

Reply:
Sure, let me update it.